draft-thomson-httpbis-http2bis.xml

```diff
@@ -3482,8 +3482,9 @@
         </t>
         <t>
           The TLS implementation MUST support the <xref target="TLS-EXT">Server Name Indication
-          (SNI)</xref> extension to TLS. HTTP/2 clients MUST indicate the target domain name when
-          negotiating TLS.
+          (SNI)</xref> extension to TLS.  If the server is identified by a DNS name, clients MUST
```

Comment:
Is "DNS name" the right terminology here? 3986 uses [reg-name](https://tools.ietf.org/html/rfc3986#section-3.2.2).

Reply:
I think so.  `reg-name` is a broad umbrella that covers a bunch of things that might include other name resolution systems (NetBIOS, appletalk, /etc/hosts, namecoin, onion names).  And RFC 6066 uses "host name", which implies that it is also generic.  We could use "host name" instead, I guess.  That would be more generic, but it also implies a stronger requirement that might not be appropriate for those other uses.  I'm certain about DNS names, but I can't say for sure about the others.

Reply:
I think your reasoning is sound. Is 'DNS name' formally defined somewhere? If so, we should probably reference.

Reply:
I found something in RFC 8499 that seems right, but it means changing to "domain name".  I'll propose the same for QUIC, where this text comes from originally.